models/logs_gather_cmd_request.go

```diff
@@ -0,0 +1,215 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package models
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// LogsGatherCmdRequest logs gather cmd request
+//
+// swagger:model logs_gather_cmd_request
+type LogsGatherCmdRequest struct {
```

Comment:
This is great, but how are you planning to deliver it? it seems that in order to do unmarshel in the agent side the agent will need to import this version of the service, I'm unsure you can deliver this change independently.